oximeter/oximeter/schema/switch-rib.toml

```diff
@@ -0,0 +1,30 @@
+format_version = 1
+
+[target]
+name = "switch_rib"
+description = "Maghemite router routing information base"
+authz_scope = "fleet"
+versions = [
+    { version = 1, fields = [ "hostname", "link_id", "rack_id" ] }
+]
+
+[[metrics]]
+name = "active_routes"
+description = "The number of currently active routes"
+units = "count
```

Comment:
seem to be missing a closing quote here

Reply:
Thanks, fixed in 2823506b00d48c661911faca9781c50e5ed35377